src/results.rs

```diff
@@ -1,11 +1,11 @@
-use crate::config::SuiteMetadata;
+use crate::config::PlanMetadata;
 use crate::section::{WritePiggybackSection, WriteSection};
 use camino::{Utf8Path, Utf8PathBuf};
 use serde::Serialize;
 use std::collections::HashMap;
 
-pub fn suite_results_directory(results_directory: &Utf8Path) -> Utf8PathBuf {
-    results_directory.join("suites")
+pub fn plan_results_directory(results_directory: &Utf8Path) -> Utf8PathBuf {
+    results_directory.join("plans")
```

Comment:
This is probably an incompatible change. What are users expected to see/do?

Reply:
I will write a werk in Checkmk about this. Users will have uninstall the agent and delete the Robotmk output directory, that is probably the safest way. I think before the release we can still do this, afterwards, we would have to at least consider this in more detail.